static/styles/settings.css

```diff
@@ -197,7 +197,8 @@ td .button {
     border-bottom: 1px solid hsl(0, 0%, 86%) !important;
 }
 
-#admin-user-list .table tr:first-of-type td {
+#admin-user-list .table tr:first-of-type td,
+#admin-bot-list .table tr:first-of-type td {
     border-top: none;
 }
```

Comment:
Grepping for this ID, why didn't you also change this one? 

`static/styles/settings.css:#admin-user-list .last_active {`